mongodbatlas/resource_mongodbatlas_database_user_test.go

```diff
@@ -148,6 +148,43 @@ func TestAccResourceMongoDBAtlasDatabaseUser_withAWSIAMType(t *testing.T) {
 	})
 }
 
+func TestAccResourceMongoDBAtlasDatabaseUser_withAWSIAMType_import(t *testing.T) {
+	var (
+		dbUser       matlas.DatabaseUser
+		resourceName = "mongodbatlas_database_user.test"
+		username     = "arn:aws:iam::358363220050:user/mongodb-aws-iam-auth-test-user"
```

Comment:
this user is in our account right?

Reply:
we could just add an env variable i.e  os.Getenv("TEST_DB_USER_IAM_ARN"), and if not set just set arn:aws:iam::358363220050:user/mongodb-aws-iam-auth-test-user, in case somebody does a clean-up in the innovation account we just update the env variable.

Reply:
Got it! :-)